data/Jaideep25-tech.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Jaideep Singh",
   "type": "personal",
+  "displayStatsPublic": true,
   "bio": "Full Stack **Flutter** developer | Open Source Advocate | Freelancer | Mentor",
   "tags": [
```

Comment:
One suggestion please remove the tag `"Full Stack Cross Platform Mobile Application Development",` and add another.